test/extended/operators/olm.go

```diff
@@ -70,4 +71,19 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-24061 - [bz 1685230] OLM operator should use imagePullPolicy: IfNotPresent
+	// author: bandrade@redhat.com
+	g.It("have imagePullPolicy:IfNotPresent on thier deployments", func() {
+		var deploymentResource = [3]string{"catalog-operator", "olm-operator", "packageserver"}
```

Comment:
nit: since this is scoped to a function you can use a short declaration (also a slice instead of an array)

```golang
deploymentResources := []string{"catalog-operator", "olm-operator", "packageserver"}
```

Reply:
Changed from array to slice, thanks for your suggestion